templates/operations.html

```diff
@@ -41,16 +41,21 @@ <h2>Operations</h2>
                             </div>
                         </div>
                         <div>&nbsp;&nbsp;</div>
-                        <button x-bind:disabled="!(searchInput && filteredOperations.length === 0)" type="button"
+                        <button x-bind:disabled="!(searchInput && filteredOperations.length === 0)"
                                 title="creates a new operation based on the last created operation settings "
                                 class="button is-primary is-small"
                                 x-on:click="addOperation(true, startOperation)"><i class="pr-1 fas fa-plus"></i>
```

Comment:
Sonarcube doesn't like `<i>` tags for some reason, so could you replace all instances of it in this file with `<em>` instead?